structuredefinitions/UKCore-AllergyIntolerance.xml

```diff
@@ -88,6 +79,28 @@
         <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Organization" />
       </type>
     </element>
+    <element id="AllergyIntolerance.clinicalStatus.extension">
+      <path value="AllergyIntolerance.clinicalStatus.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="0" />
+    </element>
+    <element id="AllergyIntolerance.clinicalStatus.extension:myExtension">

```

Comment:
Slice name is "myExtension" here (also two lines below).  Should be allergyIntoleranceEnd if following previous naming.

Reply:
Thanks, changed.